tests/source/configs/group_imports/One.rs

```diff
@@ -2,7 +2,7 @@
 use chrono::Utc;
 use super::update::convert_publish_payload;
 
-use juniper::{FieldError, FieldResult};
+pub use juniper::{FieldError, FieldResult};
```

Comment:
Just taking a quick glance at the tests, and was wondering why an existing test changed?